test/Microsoft.AspNetCore.Sockets.Tests/HttpConnectionDispatcherTests.cs

```diff
@@ -57,6 +57,39 @@ public HttpConnectionDispatcherTests(ITestOutputHelper output) : base(output)
             }
         }
 
+        [Fact]
+        public async Task CheckThatThresholdValuesAreEnforced()
```

Comment:
There should also be a test that sends data via /send to make sure it doesn't return until back pressure is released.